pkg/scheduler/core/generic_scheduler.go

```diff
@@ -1121,7 +1136,7 @@ func selectVictimsOnNode(
 	// inter-pod affinity to one or more victims, but we have decided not to
 	// support this case for performance reasons. Having affinity to lower
 	// priority pods is not a recommended configuration anyway.
-	if fits, _, err := podFitsOnNode(pod, meta, nodeInfoCopy, fitPredicates, queue, false); !fits {
+	if fits, _, err, _ := podFitsOnNode(pluginContext, pod, meta, nodeInfoCopy, fitPredicates, queue, false, filterPluginsFunc); !fits {
```

Comment:
I think the following is correct since the err is the last return value

```suggestion
	if fits, _, _, err := podFitsOnNode(pluginContext, pod, meta, nodeInfoCopy, fitPredicates, queue, false, filterPluginsFunc); !fits {
```

Reply:
done